common/src/chain/tokens/rpc.rs

```diff
@@ -63,18 +136,23 @@ pub struct RPCFungibleTokenInfo {
     pub metadata_uri: Vec<u8>,
     pub circulating_supply: Amount,
     pub total_supply: RPCTokenTotalSupply,
-    pub is_frozen: bool,
+    pub is_locked: bool,
+    pub frozen: RPCIsTokenFrozen,
+    pub authority: Option<Destination>,
```

Comment:
not sure why is it optional? Is it because we still support v0 fungible tokens somewhere in the wallet?

Reply:
because of the old v0 tokens

Reply:
I'm completely OK with removing support for v0.

Reply:
I thought we dropped them from the wallet

Reply:
removed